dockerfiles/web/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG MAKE_PYTHON_VERSION
 ARG DOCKER_BUILD_ENV
-FROM python:${MAKE_PYTHON_VERSION:-3.11}-buster
+FROM python:${MAKE_PYTHON_VERSION:-3.11}-bullseye
```

Comment:
We use `slim-bullseye` for our production build, but that didn't seem to work for the test build. The Seattle instance uses `bullseye` so we should be golden.